app/helpers/project_helper.rb

```diff
@@ -16,11 +16,9 @@ def mail_to_path(email)
   end
 
   def converting_on_date(project)
-    conversion_date = project.conversion_date
+    return project.conversion_date.to_formatted_s(:govuk) unless project.conversion_date.nil?
 
-    return conversion_date.date.to_formatted_s(:govuk) unless conversion_date.provisional?
-
-    date = conversion_date.date.to_formatted_s(:govuk)
+    date = project.provisional_conversion_date.to_formatted_s(:govuk)
     tag = govuk_tag(text: "provisional", colour: "grey")
 
     "#{date} #{tag}".html_safe
```

Comment:
This hasn't been introduced in this commit but I'm wondering why this method either returns a formatted date, or an HTML tag containing the date. Line 19 returns a formatted date if the `project.conversion_date` is not nil, and otherwise returns `"#{date} #{tag}".html_safe` which is an HTML snippet. Seems odd that it returns two different things?

Reply:
Yeah, it sucks! I'll make a note to come back to it as I work through more conversion date and slipping stuff.

Reply:
Second thought! This is a helper and it returns a string that gets rendered into a view - either jsut a date, or a date with the 'provisional' tag next to it, you could argue it does return the same thing - but I will try and make it a bit better if I can!?